symfony/client/src/core/util/validation/rules.ts

```diff
@@ -356,6 +356,18 @@ export const startDateShouldBeBeforeEndDate = (
   };
 };
 
+export const maxCurrency = function(maxValue: number) {
```

Comment:
add unit tests for this,
See https://github.com/orangehrm/orangehrm/blob/5.0-beta/symfony/client/src/core/util/validation/__tests__/rules.spec.ts